java/code/src/com/suse/manager/xmlrpc/iss/IssHandler.java

```diff
@@ -133,33 +131,30 @@ public int storeAccessToken(User loggedInUser, String fqdn, String token) {
     }
 
     /**
-     * Registers automatically a remote server with the specified ISS role.
+     * Registers automatically a remote PERIPHERAL server.
      * @param loggedInUser the user logged in. It must have the sat admin role.
      * @param fqdn the FQDN of the remote server to register
-     * @param role the ISS role of the remote server. Can be either HUB or PERIPHERAL
      * @param username the name of the user, needed to access the remote server. It must have the sat admin role.
      * @param password the password of the user, needed to access the remote server.
      * @return 1 on success, exception otherwise
      *
      * @apidoc.doc Registers automatically a remote server with the specified ISS role.
      * @apidoc.param #session_key()
      * @apidoc.param #param_desc("string", "fqdn", "the FQDN of the remote server to register ")
-     * @apidoc.param #param_desc("string", "role", "the ISS role of the remote server. Either HUB or PERIPHERAL")
      * @apidoc.param #param_desc("string", "username", "the name of the user, needed to access the remote server
      * It must have the sat admin role")
      * @apidoc.param #param_desc("string", "password", "the password of the user, needed to access the remote
      * server")
      * @apidoc.returntype #return_int_success()
      */
-    public int register(User loggedInUser, String fqdn, String role, String username, String password) {
-        return register(loggedInUser, fqdn, role, username, password, null);
+    public int register(User loggedInUser, String fqdn, String username, String password) {
```

Comment:
@mackdk : You renamed the internal API to `registerHub`. I wonder if we should rename this to `registerPeripheral` ?

Reply:
Yeah, I was thinking that too while doing the change. I think it would be a good idea.

Reply:
:+1: on it